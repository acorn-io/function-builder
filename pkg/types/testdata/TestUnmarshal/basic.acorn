targets: dev: {
	from:       "required"
	shell:      "/bin/sh"
	entrypoint: `echo "one two" three`
	workdir:    "wd"
	cacheVolumes: ["foo"]
	stopSignal: "stop"
	run: [{
		command: "echo"
	}]
}
