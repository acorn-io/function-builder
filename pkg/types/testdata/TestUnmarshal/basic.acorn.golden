{
  "targets": {
    "dev": {
      "from": "required",
      "shell": [
        "/bin/sh"
      ],
      "stopSignal": "stop",
      "cacheVolumes": [
        "foo"
      ],
      "entrypoint": [
        "echo",
        "one two",
        "three"
      ],
      "workdir": "wd",
      "run": [
        {
          "command": "echo"
        }
      ]
    }
  }
}